<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerItemWidgetClass</class>
 <widget class="QWidget" name="PlayerItemWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>85</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>85</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>85</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PlayerItemWidget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="logoPlayer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelPlayerIndex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Player 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decScore">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border-style: none;
image: url(:/Versus/arrow-left.png);
}
QPushButton:hover
{
image: url(:/Versus/arrow-left-h.png);
}
QPushButton:pressed
{
image: url(:/Versus/arrow-left-p.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="incScore">
   <property name="geometry">
    <rect>
     <x>461</x>
     <y>20</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border-style: none;
image: url(:/Versus/arrow-right.png);
}
QPushButton:hover
{
image: url(:/Versus/arrow-right-h.png);
}
QPushButton:pressed
{
image: url(:/Versus/arrow-right-p.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>400</width>
     <height>5</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="logoTeam">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border:0px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="editScore">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>35</pointsize>
     <bold>true</bold>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="maxLength">
    <number>10</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="editName">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="editTeam">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="favPlayers">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>32</width>
     <height>32</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
image: url(:/Versus/expand.png);
border:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="favTeams">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
image: url(:/Versus/expand_small.png);
border:0px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>logoPlayer</tabstop>
  <tabstop>editName</tabstop>
  <tabstop>logoTeam</tabstop>
  <tabstop>editTeam</tabstop>
  <tabstop>favPlayers</tabstop>
  <tabstop>favTeams</tabstop>
  <tabstop>editScore</tabstop>
  <tabstop>decScore</tabstop>
  <tabstop>incScore</tabstop>
 </tabstops>
 <resources>
  <include location="../RC/Versus.qrc"/>
 </resources>
 <connections/>
</ui>
